tests/test_wave_01.py

```diff
@@ -15,11 +15,11 @@ def test_create_successful_movie():
     new_movie = create_movie(movie_title, genre, rating)
 
     # Assert
-    assert new_movie["title"] is MOVIE_TITLE_1
-    assert new_movie["genre"] is GENRE_1
+    assert new_movie["title"] == MOVIE_TITLE_1
+    assert new_movie["genre"] == GENRE_1
```

Comment:
Great call in changing the asserts to use `==` instead of `is`. `==` checks for value equivalency while `is` checks for value AND reference equivalency. `is` is typically used when we check whether a value equates to `None` i.e. "if user_data is None".